Agent/Harvester/NRMAHarvesterConnection.h

```diff
@@ -13,13 +13,16 @@
 #import "NRMAConnection.h"
 #import "NRMAOfflineStorage.h"
 
+#define kCOLLECTOR_CONNECT_URI         @"/mobile/v5/connect"
+#define kCOLLECTOR_DATA_URL            @"/mobile/v3/data"
```

Comment:
These are also defined in Constants.h. Is it possible to consolidate the two uses of these down to one, either here or there?